sbws/lib/v3bwfile.py

```diff
@@ -827,17 +848,25 @@ def bw_torflow_scale(bw_lines, desc_bw_obs_type=TORFLOW_OBS_MEAN,
             # See https://trac.torproject.org/projects/tor/ticket/8494
             # just applying the formula above:
             desc_bw = min(desc_bw_obs, l.desc_bw_bur, l.desc_bw_avg)
+            if l.cons_is_unmeasured:
+                min_desc_cons_bw = desc_bw
+            # If the relay is measured, use the minimum between the descriptors
+            # bandwidth and the consensus bandwidth
+            # The consensus bandwidth in a measured relay has been obtained
+            # doing the same calculation as here
+            else:
+                min_desc_cons_bw = min(desc_bw, l.cons_bw)
             bw_new = kb_round_x_sig_dig(
```

Comment:
Each step is still in a line

Reply:
There are 15 different functions, variables, and operators used to calculate bw_new.
But most people can only understand 5-9 things at once:
https://en.m.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two

Please split this calculation into different calculations, so that each calculation has no more than 7 functions, variables, and operators.